admin/src/pages/Order/components/OrderDetails/index.js

```diff
@@ -0,0 +1,93 @@
+import React, { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+import OrderDetailTable from "./orderDetailTable";
+import { useDispatch, useSelector } from "react-redux";
+import { Steps, Select } from "antd";
+import "antd/dist/antd.css";
+import { getOrderDetailRequest } from "../../../../actions";
+import Layout from "../../../../components/Layout";
+import FormatDate from "../../../../components/UI/FormatDate";
+
+import "./style.scss";
+
+const { Option } = Select;
+const { Step } = Steps;
+
+export default function OrderDetails() {
+  let { orderId } = useParams();
+  const { orderDetails, loading } = useSelector((state) => state.orders);
+  const [currentStep, setCurrentStep] = useState(1);
+
+  const dispatch = useDispatch();
+  const step = (status) => {
+    if (status == "ordered") return 0;
+    if (status == "in_progress") return 1;
+    if (status == "completed") return 2;
+  };
+
+  useEffect(() => {
+    dispatch(getOrderDetailRequest(orderId));
+  }, []);
+
+  useEffect(() => {
+    setCurrentStep(step(orderDetails.status));
+  }, [orderDetails.status]);
+
+  const handleChange = (value) => {
+    // dispatch(updateOrdresRequest({ orderId: selectedOrder._id, type: value }));
+    console.log(value);
+  };
+
+  if (loading) {
+    return (
+      <Layout sidebar>
+        <h3>Wait</h3>
+      </Layout>
+    );
+  }
+
+  return (
+    <Layout sidebar>
+      <div className='order-detail'>
+        <h3 className='order-owner'>
+          Order by :
+          <span>
+            <strong>{orderDetails.user.userName}</strong>
+          </span>
+        </h3>
+        <OrderDetailTable data={orderDetails.items} />
+        <h4>
+          Total Price: <span>{orderDetails.totalAmount}</span>
+        </h4>
+
+        <Steps current={currentStep}>
+          {orderDetails.orderStatus.map((status) => {
+            return (
+              <Step
```

Comment:
arrow function nếu mà k có if else hay gì thì return thẳng cho ngắn gọn nhé
```suggestion
          {orderDetails.orderStatus.map((status) => (
              <Step
```